ert_shared/plugins/hook_specifications/visualization.py

```diff
@@ -0,0 +1,8 @@
+from ert_shared.plugins.plugin_manager import hook_specification
+
+
+@hook_specification
+def register_visualization_plugin(handler):
+    """
+    launch visualization solution for ERT
+    """
```

Comment:
Could you describe the expected format of the response eg. that we expect a class with name and run() to be added using the add_plugin?

Reply:
:+1: